app/views/ops/_tenant_form.html.haml

```diff
@@ -1,54 +1,13 @@
 - @angular_form = true
 
-%form.form-horizontal#form_div{"name"          => "angularForm",
-                               "ng-controller" => "tenantFormController as vm",
-                               "miq-form"      => "true",
-                               "model-copy"    => 'vm.modelCopy',
-                               "model"         => 'vm.tenantModel',
-                               'form-changed'  => true,
-                               "ng-show"       => "vm.afterGet"}
+#form_div
+
   = render :partial => "layouts/flash_msg"
-  %div
-    %div
-      .form-group{"ng-class" => "{'has-error': angularForm.name.$invalid}"}
-        %label.col-md-2.control-label{"for" => "tenant_name"}
-          = _("Name")
-        .col-md-8
-          %input.form-control{"type"        => "text",
-                              "id"          => "tenant_name",
-                              "name"        => "name",
-                              "ng-model"    => "vm.tenantModel.name",
-                              "ng-disabled" => "vm.tenantModel.default && vm.tenantModel.use_config_for_attributes",
-                              "maxlength"   => "#{MAX_NAME_LEN}",
-                              "miqrequired" => ""}
-          %span.help-block{"ng-show" => "angularForm.name.$error.miqrequired"}
-            = _("Required")
-      .form-group{"ng-class" => "{'has-error': angularForm.description.$invalid}"}
-        %label.col-md-2.control-label{"for" => "tenant_description"}
-          = _("Description")
-        .col-md-8
-          %input.form-control{"type"        => "text",
-                              "id"          => "tenant_description",
-                              "name"        => "description",
-                              "ng-model"    => "vm.tenantModel.description",
-                              "maxlength"   => "#{MAX_DESC_LEN}",
-                              "miqrequired" => ""}
-            %span.help-block{"ng-show" => "angularForm.description.$error.miqrequired"}
-              = _("Required")
-      .form-group{"ng-show" => "vm.tenantModel.default"}
-        %label.col-md-2.control-label
-          = _("Use Configuration Settings")
-        .col-md-10
-          %input#use_config{"bs-switch"       => "",
-                            "type"            => "checkbox",
-                            "name"            => "use_config_for_attributes",
-                            "ng-model"        => "vm.tenantModel.use_config_for_attributes",
-                            "switch-on-text"  => "Yes",
-                            "switch-off-text" => "No"}
-      %input{"type" => "hidden", "name" => "divisible", "value" => "{{vm.tenantModel.divisible}}"}
-  = render :partial => "layouts/angular/generic_form_buttons"
+
+  %tenant-component{:id            => @tenant.id,
```

Comment:
Yes, I can rename `id` appropriately.

> (Also, I wonder if we should be doing miq_bootstrap('tenant-component') instead of #form_div..)

I did try that, but IIRC, it did not work.
But let me re-confirm.